action.yml

```diff
@@ -17,6 +17,14 @@ inputs:
     description: 'the location of the source repository'
     required:    false
     default:     'https://gitlab.freedesktop.org/gstreamer/gstreamer.git'
+  buildSource:
+    description: 'whether to use installer package or build from source'
+    required: false
+    default: false

```

Comment:
I realize if someone is already using this on Linux (like us in the S3 plugin project), this will break for them. Maybe rename this to `forceBuildFromSource`. If it's true on macOS we can still fail, on Windows the existing check makes sense, then on Linux we don't even check.